storybook/stories/API/tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { pageData } from '../../data';
+import { Line, LineChart, ResponsiveContainer, Tooltip } from '../../../../src';
+
+export default {
+  component: LineChart,
+};
+
+export const SimpleTooltip = {
+  render: (tootipArgs: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <LineChart {...tootipArgs}>
+          <Line dataKey="uv" />
+          <Tooltip />
+        </LineChart>
```

Comment:
```suggestion
        <LineChart data={pageData} >
          <Line dataKey="uv" />
          <Tooltip  {...tootipArgs} />
        </LineChart>
```
The args must still be passed to the Tooltip component.

Reply:
And the chart needs data.

Reply:
Thank you, I really made a massive mistake there. Working on it!